src/QuantumSdk/Sdk/Sdk.targets

```diff
@@ -101,10 +101,15 @@
   <!-- Invokes the Q# command line compiler to build the project. -->
   <Target Name="QsharpCompile" 
           Condition="'$(DesignTimeBuild)' != 'true'"
-          Inputs="@(QsharpCompile);@(QscCommandArgsFile)"
+          Inputs="@(QscCommandArgsFile)"
           Outputs="$(GeneratedFilesOutputPath)$(PathCompatibleAssemblyName).bson"
           DependsOnTargets="PrepareQsharpCompile"
           BeforeTargets="PrepareCsharpCompileAfterCsharpGeneration">
+    <ItemGroup>
+      <FilesToRemove Include="$(GeneratedFilesOutputPath)**/*.g.cs" />
```

Comment:
maybe use `_FilesToRemove` to make sure it doesn't collide with some other target?

Reply:
agreed.